docs/lib/Home/index.js

```diff
@@ -100,7 +100,7 @@ npm install --save reactstrap react-addons-transition-group react-addons-css-tra
             <p>Check out the demo <a href="http://output.jsbin.com/dimive/latest">here</a></p>
```

Comment:
Looks like this link no longer works. Need to figure out replacement.

I know this is carry over from existing docs not related to popper but I remember see it .. Might as well either remove it.

Reply:
We'll have to update this later, after v5 so that the example can include v5 things. I am thinking that we should use webpackbin or codesandbox.